code/__DEFINES/components.dm

```diff
@@ -411,6 +411,7 @@
 #define COMSIG_TRY_STORAGE_QUICK_EMPTY "storage_quick_empty"			//(loc) - returns bool - if loc is null it will dump at parent location.
 #define COMSIG_TRY_STORAGE_RETURN_INVENTORY "storage_return_inventory"	//(list/list_to_inject_results_into, recursively_search_inside_storages = TRUE)
 #define COMSIG_TRY_STORAGE_CAN_INSERT "storage_can_equip"				//(obj/item/insertion_candidate, mob/user, silent) - returns bool
+#define COMSIG_RETURN_MAX_COMBINED_W "storage_max_combined_weight"		//() - returns number.

```

Comment:
No number returns in signals allowed, you may only return flags

Reply:
https://github.com/tgstation/tgstation/blob/9e34b3d6a15df071938932f46e5276eb9529ccab/code/__DEFINES/dcs/signals.dm#L349 ?

Reply:
Just because other things do it doesn't mean it acceptable